### PR TITLE
Update publish-uwp-app-store.md

### PR DESCRIPTION
**Pull request by DavidShoe**
Update docs on platform support requirements noting we do not support arm32 bit.  But do support 32 / 64 for non arm.

### CODE REVIEW

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/how-to/publish-uwp-app-store.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The Microsoft Store requires submitting a packaged app.  Package your app as fol
 
 1. Associate the application with the reserved name, by selecting **Project** > **Publish** > **Associate App with the Store**.
 
-1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Only 64-bit versions of the WebView2 Runtime will work in UWP:
+1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Note: 32-bit is not supported for Arm due to dependencies on Edge which does not support Arm32. 32/64-bit versions of the WebView2 Runtime will work in UWP on non Arm platform:
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/publish-uwp-app-store.md:47]: global: ARM allcaps.  
Microsoft Edge (add Microsoft)

Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/publish-uwp-app-store.md:47]:
```suggestion
1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Note: 32-bit is not supported for ARM, due to dependencies on Microsoft Edge, which does not support ARM32. 32/64-bit versions of the WebView2 Runtime will work in UWP on non ARM platforms:
```

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/publish-uwp-app-store.md:47]: Are the non-supported configurations listed here in this UI, or not?

File: microsoft-edge/webview2/how-to/publish-uwp-app-store.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The Microsoft Store requires submitting a packaged app.  Package your app as fol
...
 
    ![Package Selection](publish-uwp-app-store-images/package-selection.png)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/publish-uwp-app-store.md:49]:
```suggestion
1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations:

   ![Package Selection](publish-uwp-app-store-images/package-selection.png)

   32-bit and 64-bit versions of the WebView2 Runtime will work in UWP on non-ARM platforms.  32-bit is not supported for ARM, due to dependencies on Microsoft Edge, which doesn't support ARM32.
```

File: microsoft-edge/webview2/how-to/publish-uwp-app-store.md [hunk 3]
```diff
@@ -44,9 +44,11 @@ The Microsoft Store requires submitting a packaged app.  Package your app as fol
 
 1. Associate the application with the reserved name, by selecting **Project** > **Publish** > **Associate App with the Store**.
 
-1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Only 64-bit versions of the WebView2 Runtime will work in UWP:
+1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations:
 
-   ![Package Selection](publish-uwp-app-store-images/package-selection.png)
+   ![Selecting target configurations](publish-uwp-app-store-images/package-selection.png)
+
+   32-bit and 64-bit versions of the WebView2 Runtime will work in UWP on non-ARM platforms.  32-bit is not supported for ARM, due to dependencies on Microsoft Edge, which doesn't support ARM32.
```
Comment (JasonAndrewWriter) [inline microsoft-edge/webview2/how-to/publish-uwp-app-store.md:51]: 32-bit and 64-bit versions of the WebView2 Runtime will work in UWP on non-ARM platforms.  32-bit is not supported for ARM, due to dependencies on Microsoft Edge.

**Review by JasonAndrewWriter — CHANGES_REQUESTED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]
Made the last changes

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

**Pull request merged**